docs/installation/linux.md

```diff
@@ -55,7 +55,7 @@ cd HELICS
 mkdir build
 cd build
 CMake ../
-cCMake . # optional, to change install path or other configuration settings
+cmake . # optional, to change install path or other configuration settings
```

Comment:
```suggestion
cmake . # optional, to change install path or other configuration settings
```

Should the comment be expanded to say how the configs can be changed (editing CMakeCache.txt file, providing -D arguments to this command invocation)? I think the idea behind using `ccmake` was that it brings up a command line interface that lists the available options and their current settings from the CMakeCache.txt file.